docs/overriding_plan_settings.rst

```diff
@@ -0,0 +1,97 @@
+    "aggregator.settings.db_store_rounds": 2,
+    "aggregator.settings.init_state_path": "save/init.pbuf",
+    "aggregator.settings.last_state_path": "save/last.pbuf",
+    "aggregator.settings.rounds_to_train": 10,
+    "aggregator.settings.write_logs": true,
+    "aggregator.template": "openfl.component.Aggregator",
+    "assigner.settings.task_groups.0.name": "train_and_validate",
+    "assigner.settings.task_groups.0.percentage": 1.0,
+    "assigner.settings.task_groups.0.tasks.0": "aggregated_model_validation",
+    "assigner.settings.task_groups.0.tasks.1": "train",
+    "assigner.settings.task_groups.0.tasks.2": "locally_tuned_model_validation",
+    "assigner.template": "openfl.component.RandomGroupedAssigner",
+    "collaborator.settings.db_store_rounds": 1,
+    "collaborator.settings.delta_updates": false,
+    "collaborator.settings.opt_treatment": "RESET",
+    "collaborator.template": "openfl.component.Collaborator",
+    "data_loader.settings": {},
+    "data_loader.template": "openfl.federated.DataLoader",
+    "network.settings.agg_addr": "auto",
+    "network.settings.agg_port": "auto",
+    "network.settings.cert_folder": "cert",
+    "network.settings.client_reconnect_interval": 5,
+    "network.settings.disable_client_auth": false,
+    "network.settings.hash_salt": "auto",
+    "network.settings.tls": true,
+    "network.template": "openfl.federation.Network",
+    "task_runner.settings": {},
+    "task_runner.template": "openfl.federated.task.task_runner.CoreTaskRunner",
+    "tasks.settings": {}
+
+
+Use **override_config** with FL_experiment.start to make any changes to the default plan settings. It's essentially a dictionary with the keys corresponding to plan parameters along with the corresponding values (or list of values). Any new key entry will be added to the plan and for any existing key present in the plan, the value will be overrriden.
+
+
+.. code-block:: python
+
+    fl_experiment.start(model_provider=MI, 
+                   task_keeper=TI,
+                   data_loader=fed_dataset,
+                   rounds_to_train=5,
+                   opt_treatment='CONTINUE_GLOBAL',
+                   override_config={'aggregator.settings.db_store_rounds': 1, 'compression_pipeline.template': 'openfl.pipelines.KCPipeline', 'compression_pipeline.settings.n_clusters': 2})
+
+
+Since 'aggregator.settings.db_store_rounds' is a field already present in the plan, its value gets replaced with the new value specified. Fields  'compression_pipeline.template' and 'compression_pipeline.settings.n_clusters' are new enteries that get added to the plan:
+
+.. code-block:: python
+
+    INFO     Updating aggregator.settings.db_store_rounds to 1...                                                                           native.py:102
+
+    WARNING  Did not find compression_pipeline.template in config. Make sure it should exist. Creating...                                   native.py:105
```

Comment:
Thanks Mansi for the extended documentation. The base plan ([here](https://github.com/intel/openfl/blob/develop/openfl-workspace/workspace/plan/plans/default/base_plan_interactive_api.yaml)) should be modified to include the default compression pipeline information (https://github.com/intel/openfl/blob/a018c4423c4d2ef5a929406912373a47e95c4183/openfl-workspace/default/plan/plan.yaml#L35-L36)

That will make it more clear for this example (and any compression pipeline) which values in the plan to modify.

We should also change the warning to an info message, because it will be somewhat common for custom fields to be passed into the plan